pkg/v1/client.go

```diff
@@ -188,6 +188,10 @@ func (result *ResponseResult) extractErr() error {
 	}
 	defer result.Body.Close()
 
+	if len(body) == 0 {
+		result.Err = fmt.Errorf("mks-go: got the %d status code from the server", result.StatusCode)
```

Comment:
Not sure if this error message about response status code corresponds with actual situation when response body is empty.
Something like "Empty response from the server" is better in my book.